lib/cable_ready.rb

```diff
@@ -17,6 +17,8 @@
 require "cable_ready_helper"
 
 module CableReady
+  autoload :MemoryCacheDebounceAdapter, "cable_ready/updatable/memory_cache_debounce_adapter"
```

Comment:
Is there any reason why this is using `autoload` over a regular `require`?

Reply:
the app at hand might not use `Updatable`, which is itself autoloaded by living in `app/`

Reply:
also, in the future there will probably be more than one adapter which you can specify via config. then even more it wouldn't make sense to preemptively load all of them.